packages/p/pep8/package.yml

```diff
@@ -1,8 +1,9 @@
 name       : pep8
 version    : 1.7.1
-release    : 10
+release    : 11
 source     :
     - https://github.com/PyCQA/pycodestyle/archive/1.7.1.tar.gz : ac48ac8a27b7d184a698bfc1ba1c441dfeeac359979d71a23a9eebc131d0c6fc
+homepage   : https://pycodestyle.pycqa.org/en/latest/
 license    : Expat
```

Comment:
`Expat` is not part of the [SPDX License List](https://spdx.org/licenses/) . But `Expat` is just another name for the `MIT` license. Please change this to `MIT`

See [here](https://en.wikipedia.org/wiki/MIT_License#License_terms)  for more explanation